index.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const bodyParser = require('body-parser')
+const mongoose = require("mongoose")
+const cors = require("cors")
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(cors())
+
+const products = require("./routes/products.router.js")
+const cart = require('./routes/cart.router.js')
+const wishlist = require('./routes/wishlist.router.js')
+
+const { initializeDBConnection } = require("./db/db.connect.js")
+
+const PORT = 3000;
```

Comment:
hosting on static port will give you an error when you try to develop it on Heroku , because from here you don't know which port heroku is going to assign

```
const PORT = process.env.PORT || 3000
```